scripts/unit_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# RUN Go unittests
+go test -v -cover ./...
```

Comment:
Here we have some problems.

`go test -v -cover ./... ` must be run under `/tmp/elasticdl` directory. However, `/tmp/elasticdl` is created at build.sh.

Thus, there is a dependency between build.sh and unit_test.sh.  It's not what we want when splitting two scripts.

How about adding a flag for build_and_test.sh:

```
WITH_TEST=$1


if [ WITH_TEST ] then;
   go test -v -cover ./...
fi

if [ WITH_TEST ] then;
  pytest elasticdl/python/tests elasticdl_preprocessing/tests --cov=elasticdl/python --cov-report=xml
  mkdir -p ./build
  mv coverage.xml ./build
fi
```

Reply:
Good idea!